### ISSUE TITLE
Close button in Subscription screen does not

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/guardian-vpn-ios#293]
TF version: 2(2.202011121142)
iPhone 6s

**Steps**
Start the app
On Subscribe screen tap on 'x' button

**Actual**
The button is clicked but nothing happens

**Expected**
That screen is closed and the main view is shown

**Comment by rlr** [comment]
This issue should be moved to https://github.com/mozilla-mobile/mozilla-vpn-client

**Comment by isabelrios** [comment]
Yeah, sorry I just asked in the channel to be sure about the correct place, I will do that. Thanks
